touchforms/backend/touchcare.py

```diff
@@ -390,6 +160,33 @@ def filter_cases(filter_expr, api_auth, session_data=None, form_context=None):
         return {'cases': filter(lambda x: x, case_list.split(","))}
     except (XPathException, XPathSyntaxException), e:
         raise TouchcareInvalidXPath('Error querying cases with xpath %s: %s' % (filter_expr, str(e)))
+    except Exception, e:
+        print "E: ", e
+        e.printStackTrace()
+
+
+def get_fixtures(filter_expr, auth, session_data=None, restore=None):
+
+    modified_xpath = "join(',', instance('products')/products/product%(filters)s/@id)" % \
+        {"filters": filter_expr}
+
+    ccInstances = CCInstances(session_data, auth, restore)
+    productsInstance = ExternalDataInstance("jr://instance/fixture/commtrack:products", "products")
+
+    try:
+        productsInstance.initialize(ccInstances, "products")
+    except (HTTPError, URLError), e:
```

Comment:
I'm guessing we aren't connecting to HQ for these anymore so we can remove this try except (same as above)